packages/codemods/src/transforms/v2/web-react/__testfixtures__/button-text.input.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+// @ts-ignore: No declaration
```

Comment:
Could you maybe put down the reasons for these TS/ESlint ignores?

Reply:
more description added